### PR TITLE
remove/replace outdated links

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#530

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/integrated-apps-and-azure-ads.md [hunk 1]
```diff
@@ -35,10 +35,9 @@ Managing Office 365 apps requires you to have knowledge of apps in Azure AD. The
...
-|[Office 365 APIs platform overview](https://go.microsoft.com/fwlink/?LinkId=617140) <br/> |The Office 365 APIs lets you provide access to your customer's Office 365 data, including the things they care about most—their mail, calendars, contacts, users and groups, files, and folders. There is a good diagram in this article that illustrates the relationship among Office 365 apps, Azure AD, and the data that the apps access.  <br/> |
+|[Office 365 Management APIs overview](https://docs.microsoft.com/office/office-365-management-api/office-365-management-apis-overview) <br/> |The Office 365 APIs lets you provide access to your customer's Office 365 data, including the things they care about most—their mail, calendars, contacts, users and groups, files, and folders. There is a good diagram in this article that illustrates the relationship among Office 365 apps, Azure AD, and the data that the apps access.  <br/> |
```
Comment (JohanFreelancer9) [inline Enterprise/integrated-apps-and-azure-ads.md]:
```suggestion
|[Office 365 Management APIs overview](https://docs.microsoft.com/office/office-365-management-api/office-365-management-apis-overview) <br/> |The Office 365 APIs let you provide access to your customer's Office 365 data, including the things they care about most—their mail, calendars, contacts, users and groups, files, and folders. There is a good diagram in this article that illustrates the relationship among Office 365 apps, Azure AD, and the data that the apps access.  <br/> |
```

File: Enterprise/integrated-apps-and-azure-ads.md [hunk 2]
```diff
@@ -35,10 +35,9 @@ Managing Office 365 apps requires you to have knowledge of apps in Azure AD. The
...
 |[Integrating Applications in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/develop/quickstart-v1-add-azure-ad-app) <br/> | Learn about applications that are integrated with Azure Active Directory, and how to register your application, understand concepts behind a registered application, and learn about branding guidelines for multi tenant applications.  <br/> |
+|[Add custom tiles to the app launcher](https://docs.microsoft.com/office365/admin/manage/customize-the-app-launcher).  <br/> |The app launcher in Office 365 that makes it easier for users to find and access their apps. This article describes the ways you as a developer can get your apps to appear in users' app launchers and also give them a single sign-on (SSO) experience using their Office 365 credentials.  <br/> |
```
Comment (JohanFreelancer9) [inline Enterprise/integrated-apps-and-azure-ads.md]:
```suggestion
|[Add custom tiles to the app launcher](https://docs.microsoft.com/office365/admin/manage/customize-the-app-launcher).  <br/> |The app launcher in Office 365 makes it easier for users to find and access their apps. This article describes the ways you as a developer can get your apps to appear in users' app launchers and also give them a single sign-on (SSO) experience using their Office 365 credentials.  <br/> |
```

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 ,
your recommendations were applied,
thank you very much!

**Comment by mypil** [comment]
@chrfox - PR has been copy edited and is ready for final review based on issue #530.

Thank you.

cc: @kenwith

**Pull request merged**